build.gradle

```diff
@@ -236,6 +236,10 @@ dependencyManagement {
       entry 'tomcat-embed-websocket'
     }
```

Comment:
we can remove whole lot block for tomcat - it has been fixed in previous versions of spring boot already